README.md

```diff
@@ -18,88 +22,6 @@ Exconf follows these basic principles:
   like trigger an *Ansible* playbook or whatever other configuration management tool for
   more complex needs.
 
-At *Magine* we use *Exconf* for defining configurations for all of our different environments in one
-place independently of what service scheduler or environment specifics are. We can use *Exconf* to
-create an *Aurora* (*Mesos*) scheduler config and deploy the service, or maybe define a *Kubernetes*
-pod config for deploying the same service using different tooling in different environment. Many
-details remain the same independently of the deployment method, and Exconf provides a way to define
-these things in single location.
-
-
-## Install Exconf
-
-Clone the source repository and build a Debian package or Python egg/wheel by calling any of:
-
-```
-make egg
-```
-
-```
-make wheel
-```
-
-```
-make deb
-```
-
-You can also just install the Python package locally by:
-
-```
-make install
-```
-
-Exconf source is compatible with Python 2.7 and Python 3.5 or later.
-
-
-## Hello-World Example
-
-After you have installed exconf CLI, and you just want to try out the basic functionality, you can
-point the tool into the "example" configuration space within the exconf source code. Assuming here
-that the source code lives in directory */code/exconf*. Try out the following calls:
-
-```
-exconf --help
-```
-
-```
-exconf -c /code/exconf/example list-services
-```
-
-```
-exconf -c /code/exconf/example list-envs
-```
-
-You can also define an environment variable `EXCONF_CONFIG_ROOT` so you don't need to give the `-c`
-flag every time.
-
-```
-export EXCONF_CONFIG_ROOT=/code/exconf/example
-```
-
-Now you should be able to do just:
-
-```
-exconf list-services
-```
-
-There is only one simplistic *hello-world* service available in example directory. Try out to get
-configurations for *hello-world*, and also executing the defined template type executable, which is
-just plain *echo* command in this case, printing out a configured text message into stdout.
-
-```
-exconf variables -s hello-world -e local
-```
-
-```
-exconf template -s hello-world -e local
-```
-
-```
-exconf execute -s hello-world -e local
-```
-
-Continue reading to learn what all of the calls actually mean.
-
```

Comment:
Why did you remove all this? I'm sure it could be made clearer and more concise, but I would like the README to have some usage example and pointing out the environment variable EXCONF_CONFIG_ROOT.